tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -576,52 +576,12 @@ def test_mpu_after_pod_restart(self):
         This test tests multipart upload after data pod restart.
         """
         LOGGER.info("STARTED: Test to verify multipart upload after data pod restart.")
```

Comment:
with DTM0 we might not support multipart so this might need skip marker if not already in.

Reply:
Skip marker is there on Blocked until F-22A is available.

Reply:
We need to chnage the skip reason that with DTM0 multipart is not supported